### PR TITLE
Correct SMIL compatibility information

### PR DESCRIPTION
**Pull request by longsonr**

### CODE REVIEW

File: api/SVGAnimateColorElement.json [hunk 1]
```diff
@@ -14,19 +14,19 @@
             "version_added": null
           },
           "edge": {
-            "version_added": null
+            "version_added": false
           },
           "edge_mobile": {
             "version_added": null
```
Comment (ExE-Boss) [inline api/SVGAnimateColorElement.json]: `false`

File: api/SVGAnimateElement.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
             "version_added": null
           },
           "edge": {
-            "version_added": null
+            "version_added": false
           },
           "edge_mobile": {
             "version_added": null
```
Comment (ExE-Boss) [inline api/SVGAnimateElement.json]: `false`

File: api/SVGAnimateMotionElement.json [hunk 3]
```diff
@@ -14,7 +14,7 @@
             "version_added": null
           },
           "edge": {
-            "version_added": null
+            "version_added": false
           },
           "edge_mobile": {
             "version_added": null
```
Comment (ExE-Boss) [inline api/SVGAnimateMotionElement.json]: `false`

File: api/SVGAnimateTransformElement.json [hunk 4]
```diff
@@ -14,7 +14,7 @@
             "version_added": true
           },
           "edge": {
-            "version_added": null
+            "version_added": false
           },
           "edge_mobile": {
             "version_added": null
```
Comment (ExE-Boss) [inline api/SVGAnimateTransformElement.json]: `false`

File: api/SVGAnimationElement.json [hunk 5]
```diff
@@ -14,10 +14,10 @@
             "version_added": false
           },
           "edge": {
-            "version_added": true
+            "version_added": false
           },
           "edge_mobile": {
-            "version_added": true
+            "version_added": null
```
Comment (ExE-Boss) [inline api/SVGAnimationElement.json]: `false`

File: api/SVGAnimationElement.json [hunk 6]
```diff
@@ -55,41 +55,41 @@
...
             },
             "chrome_android": {
               "version_added": null
             },
             "edge": {
-              "version_added": null
+              "version_added": false
             },
             "edge_mobile": {
               "version_added": null
```
Comment (ExE-Boss) [inline api/SVGAnimationElement.json]: `false`

File: api/SVGAnimationElement.json [hunk 7]
```diff
@@ -103,41 +103,41 @@
...
```
Comment (ExE-Boss) [inline api/SVGAnimationElement.json]: `false`

File: api/SVGAnimationElement.json [hunk 8]
```diff
@@ -151,41 +151,41 @@
...
```
Comment (ExE-Boss) [inline api/SVGAnimationElement.json]: `false`

File: api/SVGAnimationElement.json [hunk 9]
```diff
@@ -199,34 +199,34 @@
...
```
Comment (ExE-Boss) [inline api/SVGAnimationElement.json]: `false`

File: api/SVGSetElement.json [hunk 10]
```diff
@@ -14,7 +14,7 @@
             "version_added": null
           },
           "edge": {
-            "version_added": null
+            "version_added": false
           },
           "edge_mobile": {
             "version_added": null
```
Comment (ExE-Boss) [inline api/SVGSetElement.json]: `false`

File: api/SVGAnimateColorElement.json [hunk 11]
```diff
@@ -14,19 +14,19 @@
...
```
Comment (ExE-Boss) [inline api/SVGAnimateColorElement.json]: `true`

File: api/SVGAnimateElement.json [hunk 12]
```diff
@@ -14,7 +14,7 @@
...
```
Comment (ExE-Boss) [inline api/SVGAnimateElement.json]: `true`

File: api/SVGAnimateMotionElement.json [hunk 13]
```diff
@@ -14,7 +14,7 @@
...
```
Comment (ExE-Boss) [inline api/SVGAnimateMotionElement.json]: `true`

File: api/SVGSetElement.json [hunk 14]
```diff
@@ -14,7 +14,7 @@
...
```
Comment (ExE-Boss) [inline api/SVGSetElement.json]: `true`

File: api/SVGAnimateColorElement.json (hunk 11 above)
Comment (jpmedley) [reply api/SVGAnimateColorElement.json]: I come up with `false` for all flavors of Chrome. What makes you think it's `true`.

Comment (ExE-Boss) [reply api/SVGAnimateColorElement.json]: Because `opera` + `opera_android` says `true`, but I wasn’t entirely sure about it.

Comment (longsonr) [reply api/SVGAnimateColorElement.json]: Chrome does not seem to support animateColor. Here's a testcase: https://bugzilla.mozilla.org/attachment.cgi?id=322937

**Comment by libbymc** [comment]
@ExE-Boss - Reviewed. All updates are correct for Edge + Edge mobile.

File: api/SVGAnimateColorElement.json (hunk 11 above)
Comment (longsonr) [reply api/SVGAnimateColorElement.json]: Opera is just Chrome with a different skin these days. It will always match.

Comment (jpmedley) [reply api/SVGAnimateColorElement.json]: You're generally right, but there are exceptions.

**Pull request merged**